web-extension/common/test/admin-ns.spec.js

```diff
@@ -2,12 +2,15 @@ import initAdminServer from '../backend/admin-ns.js'
 import { WalletCollection } from '../backend/wallets.js'
```

Comment:
mentioned in slack but thought I would chuck it in here too. Would like to see a few more tests just to exercise the positive and negative paths.. in particular these things

- sad path for admin.update passphrase to check it gives the encryption error
- happy path for admin.unlock
- sad path for admin.unlock to check it gives 'encryption not initialised'
- admin.lock

Reply:
These should be covered in `b42f872`